doc/ha_design.rst

```diff
@@ -5,15 +5,15 @@ TrinityX HA design and implementation
 Introduction
 ------------
 
-The TrinityX installation playbooks can set up the controller, or the controller pair, either as a regular stand-alone system, or as part of a High-Availability pair with failover of services between the two controllers of the pair.
+The TrinityX installation playbooks can set up the controller, or the controller pair, either as a regular stand-alone system, or as part of a High-Availability pair with failover of services between the two controllers.
```

Comment:
The original wording is somewhat vague, can be interpreted like there are 4 options of how to configure controller nodes, whereas in fact there are just two - non-HA and HA.